mep/common/tests.py

```diff
@@ -332,6 +334,110 @@ def test_dict_item(self):
         # integer value
         assert dict_item({13: 7}, 13) is 7
 
+class TestCheckboxFieldset(TestCase):
+
+    def test_render(self):
+
+        checkbox_fieldset = CheckboxFieldset(attrs={'name': 'sex'})
+        checkbox_fieldset.optgroups = Mock()
```

Comment:
Surprised you need a mock for this, is there not a better way to initialize without that?

Reply:
Asking because it seems too tightly coupled with the Django code, I'm not familiar with the specifics of `optgroups` and what it's supposed to return.

Reply:
The code for what gets parsed and passed to `optgroups` isn't terribly well documented, but the functioning of it was relatively clear. I thought it best to Mock here because for purposes of this, because we're not really testing if the Widget does the right thing -- it's all stock Django -- we're testing that the template does the right thing with complicated inputs.

The reason for the mock in specific is that `optgroups` is actually a method.

Reply:
Seemed like you could ignore optgroups and just initialize the fieldset the way you would normally? (Lack of documentation seems all the more reason to bypass it).  But fine with this if you add a comment.

Reply:
Thanks, I'll go with the documentation in this case. Time permitting I'll revisit the `Widget` logic and see if I can make this neater. So far widgets have been a challenge to customize the internals.